src/libPMacc/include/algorithms/math/doubleMath/sqrt.tpp

```diff
@@ -50,7 +50,7 @@ struct RSqrt<double>
 
     HDINLINE double operator( )(const double& value )
     {
-#if defined(_MSC_VER) && !defined(__CUDA_ARCH__)
```

Comment:
As far as I know, the one available on the host side is the function provided by CUDA, which internally distinguishes between host and device implementations. I think there is no native ```rsqrt``` without CUDA.
As ```__CUDACC__``` is defined even on the host side of a CUDA compilation, the current version already uses ```rsqrt``` on the host side.